fuel-core/src/args.rs

```diff
@@ -42,8 +42,8 @@ pub struct Opt {
     pub chain_config: String,
 
     /// Allows RPC Endpoints to arbitrarily modify the VM. Used for local development only
```

Comment:
IIUC refering to "RPC" here isn't accurate?

Reply:
Good catch, forgot to update the comment when I changed the name